app/controllers/api/v2/quotas_controller.rb

```diff
@@ -36,25 +38,31 @@ def search_service
         @search_service ||= QuotaSearchService.new(params, current_page, per_page)
       end
 
-      def per_page
-        5
-      end
+      def valid_includes
+        return DEFAULT_INCLUDES if include_params.empty?
+
+        valid_resources = include_params.select { |resource| ALLOWED_INCLUDES.include?(resource) }
 
-      def includes
-        return provided_includes if provided_includes.present?
+        if valid_resources.length < include_params.length
+          Sentry.capture_message("Error: invalid params in 'includes': #{invalid_includes}")
```

Comment:
So we're making noise for ourselves but not returning an error to the end user. Probably its easier to just return an error that specifies the user needs to use only quota_balance_events in the includes list.

Reply:
maybe we should raise an ArgumentError, I saw we do it in a few places.